data/icons/Makefile.am

```diff
@@ -19,9 +20,9 @@ private_icons = \
 	$(NULL)
 
 noinst_DATA = \
-	hicolor_apps_16x16_caja.svg \
-	hicolor_apps_22x22_caja.svg \
-	hicolor_apps_32x32_caja.svg \
+	#hicolor_apps_16x16_caja.svg \
+	#hicolor_apps_22x22_caja.svg \
+	#hicolor_apps_32x32_caja.svg \
```

Comment:
the same here

Reply:
feel free to change it any way you feel like it.